jans-config-api/plugins/scim-plugin/dependency-reduced-pom.xml

```diff
@@ -0,0 +1,346 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do we need to push dependency-reduce-pom.xml ? It can be generated at any time by shade plugin.

Reply:
No, we do not need this. Will remove this